src/Microsoft.SqlTools.ServiceLayer/ObjectManagement/ObjectTypes/Credential/CredentialInfo.cs

```diff
@@ -19,5 +19,7 @@ public class CredentialInfo : SqlObject
         public DateTime DateLastModified { get; set; }
         public DateTime CreateDate { get; set; }
         public string ProviderName { get; set; }
+        public string? Secret { get; set; }
+        public string? Url { get; set; }
```

Comment:
From my comment in the other PR, isn't the URL just the name for the S3 secret scenario? (which seems to be backed up by this not seemingly being used anywhere here)

Reply:
that makes sense